src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/AgoraStatus.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.agora.server.dataaccess
+
+import org.broadinstitute.dsde.agora.server.dataaccess.mongo.AgoraMongoClient._
+import org.broadinstitute.dsde.agora.server.dataaccess.permissions.AgoraEntityPermissionsClient._
+
+class AgoraStatus {
+  def status(): (Boolean, String) = {
+    val (up, message) = getMongoDBStatus
+    try {
+      // sql up, mongo may be up or down
+      sqlDBStatus
+      return (up, s"""{"status": "down", "mongo error": "$message" }""")
```

Comment:
returns are bad in scala. Anonymous functions are embedded all over the place so it can be hard to tell what you are returning from. In this case you just don't need it at all as the last value is the return value.